dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -3054,4 +3057,46 @@ public void deleteOtherRelation(Project project, Map<String, Object> result, Pro
 
     }
 
+    /**
+     * Batch release process definitions by code states.
+     *
+     * @param loginUser    Login user.
+     * @param projectCode  Project code.
+     * @param codeStates   Code states in JSON format.
+     * @return Result of the batch release process definitions.
+     */
+    @Override
+    @Transactional
+    public Map<String, Object> batchReleaseProcessDefinitions(User loginUser, long projectCode, String codeStates) {
+        Map<String, Object> result = new HashMap<>();
+        if (StringUtils.isEmpty(codeStates)) {
+            log.error("Parameter codeStates is empty, projectCode is {}.", projectCode);
+            putMsg(new HashMap<>(), Status.PROCESS_DEFINITION_CODES_IS_EMPTY);
```

Comment:
could you directly throw an exception instead of return hash map?https://github.com/apache/dolphinscheduler/blob/73b505f6397155ccdd007cf5fcabca9f6cc6447f/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessInstanceServiceImpl.java#L629